src/d/alexa-openwebif/source/skill.d

```diff
@@ -190,3 +213,18 @@ unittest
 	skill = new OpenWebifSkill("", "en-US");
 	assert(skill.getText(TextId.PleaseLogin) == AlexaText_en[TextId.PleaseLogin].text);
 }
+
+AlexaResult returnError(ITextManager texts)
+{
+	import std.format : format;
+	import std.random : uniform;
+	import std.conv : to;
+
+	AlexaResult result;
+	auto errorId = uniform!uint();
```

Comment:
The user gets a error card with this error ID in the card content and instructions to open a ticket at github with this reference. So it is easier for us to find the error in cloudwatch and provide better help - see pic for bette explanation ;-)
![image](https://cloud.githubusercontent.com/assets/6609371/22873964/9a7bc662-f1c3-11e6-83cc-93e3aaa4bebb.png)